### ISSUE TITLE
Exception when using a wildcard (**) route without redirectTo

### ISSUE THREAD

**Issue by JochemKuijpers** [Greentube/localize-router#41]
When using a wildcard route like the following, which is a [totally valid route](https://angular.io/docs/ts/latest/guide/router.html#!#wildcard), localize-router causes an exception (see below):

```js
{
    path: '**',
    component: NotFoundPageComponent
}
```

```
zone.js:516 Unhandled Promise rejection: Cannot read property 'split' of undefined ; Zone: <root> ; Task: Promise.then ; Value: TypeError: Cannot read property 'split' of undefined
    at ManualParserLoader.LocalizeParser.translateRoute (localize-router.parser.js:132)
    at ManualParserLoader.LocalizeParser._getTranslatePromise (localize-router.parser.js:119)
    at ManualParserLoader.LocalizeParser.translateRoutes (localize-router.parser.js:87)
    at ManualParserLoader.LocalizeParser.init (localize-router.parser.js:72)
    at localize-router.parser.js:234
...
```

A workaround is to use a `redirectTo` property on the wildcard path and set the component on this other path, like so:

```js
{
    path: '404',
    component: NotFoundPageComponent
},
{
    path: '**',
    redirectTo: '/404'
}
```

This has the side-effect that the original path cannot be found anymore in the ActivatedRoute snapshot so the NotFoundPageComponent cannot attempt to redirect the user to the appropriate page. This is what I want to do..

I feel like this can be easily fixed by changing [this line](https://github.com/Greentube/localize-router/blob/v0.6.4/src/localize-router.parser.ts#L116) into the following:

```
if (this.originalWildcard && this.originalWildcard.redirectTo) {
```

I'm not entirely sure about the implications of this change, so I've not made a pull request.

**Comment by JochemKuijpers** [comment]
Hmm.. I thought I'd at least make a pull request so you don't need to spend time looking for the line of code. 

Looking at the master code though, it seems this issue is [already resolved](https://github.com/Greentube/localize-router/blob/master/src/localize-router.parser.ts#L161), but not yet released.

Any reason why this isn't released yet? Is it possible to release my fix in the meantime as a hotfix?

**Comment by meeroslav** [comment]
@JochemKuijpers Thanks for your effort. I will release this fix in day/two.
It was solved as part of LazyLoading which was unfortunately never successfully tested with AOT on router version 3.x.
Sorry for delay

**Comment by meeroslav** [comment]
Solved in release 0.7.0
